docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  scrapper:
+    build: scrapper
+    image: bookfinder/scrapper:latest
+    deploy:
+      replicas: 2
+    ports:
+      - "${SCRAPPER_PORT}:8080"
+    environment:
+      - SPRING_PROFILES_ACTIVE=${ACTIVE_SCRAPPER_PROFILE}
+  db:
+    build: db
+    image: bookfinder/db:latest
+    environment:
+      - POSTGRES_USER=user
```

Comment:
Try to move this sensitive data to the secure place. For this moment it's enough, but keep it in mind.